src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -152,22 +160,32 @@ const ChatItemMyTextV2 = ({
     return [styles.containerAnon, styles.textContainer, paddingStyle];
   };
 
+  const getStyles = () => {
+    if (chatItem?.type === 'deleted') {
+      return styles.deletedText;
+    }
+
+    return styles.text;
+  };
+
   return (
     <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
-      <View style={handleTextContainerStyle()}>
-        {!isContinuous && (
-          <View
-            style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
-            <Text style={styles.userText}>{username}</Text>
-            <View style={styles.dot} />
-            <Text style={styles.timeText}>{time}</Text>
-          </View>
-        )}
-        {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
-        {attachments.length <= 0 && <LinkableText style={styles.text} text={message} />}
+      <ChatContextMenuView contextMenuType="MyChatContextMenu" chat={chatItem}>
+        <View style={handleTextContainerStyle()}>
+          {!isContinuous && (
+            <View
+              style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
+              <Text style={styles.userText}>{username}</Text>
+              <View style={styles.dot} />
+              <Text style={styles.timeText}>{time}</Text>
+            </View>
+          )}
+          {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
+          {attachments.length <= 0 && <LinkableText style={getStyles()} text={message} />}
 
-        {renderIcon()}
-      </View>
+          {renderIcon()}
+        </View>
+      </ChatContextMenuView>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getStyles` checks if the chat item type is 'deleted' and returns the appropriate style. This is a good approach for handling different styles based on the chat item type. However, it would be better to handle the case where `chatItem` or `chatItem.type` is undefined or null. This will prevent potential runtime errors due to trying to access properties of undefined or null objects.
```diff
  const getStyles = () => {
+    if (!chatItem || !chatItem.type) {
+      return styles.text;
+    }
    if (chatItem.type === 'deleted') {
      return styles.deletedText;
    }
    return styles.text;
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->